src/classes/services/WebSocketService.ts

```diff
@@ -1,30 +1,29 @@
-import router from '@/router'
-import Message from '@/types/Message'
 import MessageCommands from '@/types/MessageCommands'
 import MessageTypes from '@/types/MessageTypes'
+import MessageSent from '@/types/MessageSent'
+import MessageReceived from '@/types/MessageReceived'
 import File from '@/types/File'
 import ContentType from '@/types/ContentType'
 import base64ToArrayBuffer from '@/utils/helpers/base64ToArrayBuffer'
 import { Buffer } from 'buffer'
 
 export class WebSocketService {
-  listFiles = [] as File[]
-  wsOnMessageListeners: ((obj: Message) => void)[] = []
+  listFiles = [] as File[] | File | undefined
```

Comment:
This should be always an `array` or `null`. You shouldnt use an array or an object alternatively.